source/backend/api/Models/Concepts/Property/PropertyActivityModel.cs

```diff
@@ -1,23 +1,54 @@
 using System;
+using System.Collections.Generic;
 
 namespace Pims.Api.Models.Concepts
 {
     public class PropertyActivityModel : BaseAppModel
     {
+        #region Properties
+
         public long Id { get; set; }
 
-        public string ActivityTypeCode { get; set; }
+        public TypeModel<string> ActivityTypeCode { get; set; }
+
+        public TypeModel<string> ActivitySubtypeCode { get; set; }
+
+        public TypeModel<string> ActivityStatusTypeCode { get; set; }
+
+        public DateTime RequestAddedDateTime { get; set; }
```

Comment:
Will probably need to do a pass on these as well in my PR after we merge this, I think these are actually DateOnly's in the new format.

Reply:
Yeah, thats what I thought when I was writing this and saw your pr